packages/fetch-engine/scripts/test.ts

```diff
@@ -1,11 +1,11 @@
 import { download } from '../src'
-import path from 'path'
```

Comment:
Please keep the import convention of `import path`, as that's how we do it in all codebases.

Reply:
Oh okay, I edited that because it showed an error:

<img width="385" alt="image" src="https://user-images.githubusercontent.com/5013932/68950295-25582080-07bc-11ea-88be-e4d9d0f58fda.png">